xml/System.Reflection.Metadata/MetadataStreamOptions.xml

```diff
@@ -52,6 +52,7 @@
       <MemberValue>0</MemberValue>
       <Docs>
         <summary>By default, the stream is disposed when <see cref="T:System.Reflection.Metadata.MetadataReaderProvider" /> is disposed and sections of the PE image are read lazily.</summary>
+        <remarks>To be added.</remarks>
```

Comment:
Apparently the tool added these missing empty `<remarks>` sections. I'd like to think the server won't complain.